src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -469,16 +491,14 @@ public void onWorldTick(TickEvent.WorldTickEvent event)
     /**
      * Adds work orders to the {@link Colony#workManager}
      *
-     * @param level         Desired level
+     * @param level
+     *            Desired level
      */
     private void requestWorkOrder(int level)
     {
         for (WorkOrderBuild o : colony.getWorkManager().getWorkOrdersOfType(WorkOrderBuild.class))
         {
-            if (o.getBuildingLocation().equals(getID()))
-            {
-                return;
-            }
+            if (o.getBuildingLocation().equals(getID())) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)